sql/src/main/java/io/crate/analyze/OptimizeTableAnalyzedStatement.java

```diff
@@ -24,28 +24,24 @@
 
 import org.elasticsearch.common.settings.Settings;
 
-import java.util.Set;
+import java.util.List;
 
-public class OptimizeTableAnalyzedStatement implements DDLStatement {
+public class OptimizeTableAnalyzedStatement {
 
-    private final Set<String> indexNames;
+    private final List<String> tablesToOptimize;
     private final Settings settings;
 
-    public OptimizeTableAnalyzedStatement(Set<String> indexNames, Settings settings) {
-        this.indexNames = indexNames;
+    public OptimizeTableAnalyzedStatement(List<String> tablesToOptimize,
```

Comment:
Are these now fully qualified table names, or still the name of the indices? If the later I'd prefer to keep `indexNames`. If the former maybe rename to `fqTableNames` or clarify via javadocs.

Reply:
name of the indices. i'll keep it as `indexNames`.